composer/algorithms/distillation/README.md

```diff
@@ -0,0 +1,147 @@
+# 🏙️ Knowledge Distillation
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution) - [\[API Reference\]](#api-reference)
+
+`Computer Vision`, `Natural Language Processing`
+
+Knowledge Distillation is a technique used to "transfer" the knowledge of a pretrained model to another model. Typically the pretrained model (called the "teacher") is a larger and more complex model and is used to train a smaller, simpler model (called the "student"). This is typically done by training the student model to mimic the predictions of the teacher model, in addition to or instead of the ground truth labels. The hope is that the student model will learn a compressed representation of the knowledge of the teacher model, allowing it to achieve similar performance but with fewer parameters and faster inference times.
+
+KnowledgeDistillation
+The process of knowledge distillation. The teacher model makes predictions for a set of inputs, and the student model is trained to mimic these predictions. This image is from Chen et al., 2015.
+How to Use
+To use knowledge distillation, you will need to have both a teacher model and a student model. You can then train the student model by providing it with the predictions of the teacher model as "soft targets" (i.e., predicted probabilities for each class), in addition to the ground truth labels. The student model can be trained using standard supervised learning techniques, such as cross-entropy loss.
```

Comment:
Looks like this is extraneous/duplicated from below and above?

Reply:
Most of the readme needs to be re-written. I was fighting getting it to pass tests and gave up just before christmas.